control-plane/catalog/registration/registrations_controller.go

```diff
@@ -143,48 +143,9 @@ func (r *RegistrationsController) handleRegistration(ctx context.Context, log lo
 		return result
 	}
 
-	if r.Cache.aclsEnabled() {
```

Comment:
all the acl updates are done on the terminating gateway now to remove this controller racing with that one